modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductDefineExternalyOption.testcase

```diff
@@ -0,0 +1,926 @@
+@component-name = "portal-commerce"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Product Info Management";
+
+	setUp {
+		CommerceConfiguration.commerceSetUp(minium = "true");
+	}
+
+	tearDown {
+		CommerceConfiguration.commerceTearDown();
+	}
+
+	@description = "COMMERCE-12094. As a buyer user, I can view all the products with Define Externally configuration listed as option values in product details and checkout"
+	@priority = 5
+	test CanCheckoutBundledProductWithExternallyDefinedProductOptions {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
```

Comment:
Please move this line at the definition level and remove all occurrences in the tests